types/react-native/index.d.ts

```diff
@@ -667,6 +667,10 @@ type GeolocationError = {
     TIMEOUT: number;
 };
 
+interface MatrixTransform {
+    matrix: Array<number>;
```

Comment:
I can't find any source that shows that `matrix` is allowable at runtime.
Could you link to either the source that is handling this, or an example of usage? Thanks!

Reply:
For example, [this file](https://github.com/facebook/react-native/blob/master/ReactCommon/fabric/graphics/Transform.h) seems to define the API for allowable options, and it doesn't look like `matrix` would be allowed. Please let me know if I'm missing something, though.

Reply:
It's being used here:
https://github.com/facebook/react-native/blob/master/Libraries/StyleSheet/processTransform.js#L48-L50

Reply:
Great, thanks! 
Could you add a test for this? I don't know react-nnative particularly well, but it looks like [this file](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/73b16bf4233ed1ffe686186727e7d18d4124eecd/types/react-native/test/animated.tsx) would be a good place.

Reply:
We are just checking if the type system checks out, right? And not actually running any code. (I don't see anything being used anywhere)

animated.tsx seems to test the animation library which also happens to involve some transforms. I think a better place is index.tsx with a new component. 

I might as well add all the other transform properties as they aren't being tested for as well.